### PR TITLE
Remove experimental training feature

### PR DESCRIPTION
**Pull request by xd009642**
As discussed on the google groups

### CODE REVIEW

**Comment by xd009642** [comment]
@googlebot I signed it!

File: src/session.rs [hunk 1]
```diff
@@ -24,9 +23,11 @@ use std::ptr;
 pub struct SavedModelBundle {
     /// The loaded session.
     pub session: Session,
-    /// A meta graph definition as raw protocol buffer.
+    /// A meta graph definition as raw protocol buffer. This is deprecated in favour of the
+    /// deserialized type
```
Comment (adamcrume) [inline src/session.rs:27]: Nit: This is missing a final period.

**Comment by adamcrume** [comment]
The tests can also be removed from the test-all script.

File: src/session.rs (hunk 1 above)
Comment (xd009642) [reply src/session.rs]: Done :+1:

**Comment by xd009642** [comment]
Addressed both comments :+1:

**Comment by xd009642** [comment]
@adamcrume done, I've also taken in the latest changes in master to resolve the merge conflict

**Comment by adamcrume** [comment]
Thanks!

**Pull request merged**